server.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+// node modules
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+// npm modules
+const cowsay = require('cowsay');
+// app modules
+const parseBody = require('./lib/parse-body');
+const badRequest = require('./model/bad-request');
+const cowsayHTML = require('./lib/cowsay-html');
+// module constants
+const PORT = process.env.PORT || 3000;
+// module logic
+const server = http.createServer(function(req, res){
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
```

Comment:
Remember the instructions said to write a GET method to ``/`` that response with ``hello world``.